privaterelay/templates/includes/login.html

```diff
@@ -17,7 +18,7 @@
 					<img class="glocal-small-avatar" alt="Firefox Account Avatar" src="{{ avatar }}" />
 					<div class="glocal-account-info">
 						<span class="glocal-fxa-email">{{ request.user.email }}</span>
-						<a href="{{ FXAURL }}"" class="glocal-fxa-settings-link">Manage your Firefox Account</a>
+                        <a href="{% manage_fxa_url %}" class="glocal-fxa-settings-link">Manage your Firefox Account</a>
```

Comment:
nit: The indentation on this line changed from tabs to spaces.

Reply:
hrm ... we should be spaces anyway, but I hate inconsistency. I'll see if I can fix.

Reply:
Updated back to tabs for now.